src/Agent.Listener/Configuration/ConfigurationProvider.cs

```diff
@@ -263,18 +263,18 @@ public void UpdateAgentSetting(AgentSettings settings)
         {
             ArgUtil.NotNull(_machineGroupServer, nameof(_machineGroupServer));
 
-            DeploymentMachineGroup machineGroup = (await _machineGroupServer.GetDeploymentMachineGroupsAsync(projectName, machineGroupName)).FirstOrDefault();
+            var deploymentGroup = (await _machineGroupServer.GetDeploymentGroupsAsync(projectName, machineGroupName)).FirstOrDefault();
```

Comment:
yes  just waiting for RM M113 deployment to finish. Will wait till all UI component get fixed first and then I will work to make changes with agent code